docker/build/gene_sets.txt

```diff
@@ -0,0 +1,22 @@
+Prostate Cancer: AR Signaling=SOX9 RAN TNK2 EP300 PXN NCOA2 AR NRIP1 NCOR1 NCOR2
```

Comment:
Is this file specific to Docker provisioning? Or are you suggesting it as a new default?